components/salesforce_rest_api/sources/common.mjs

```diff
@@ -13,28 +13,10 @@ export default {
       },
     },
     objectType: {
-      type: "string",
-      label: "Object Type",
-      description: "The type of object for which to monitor events",
-      async options(context) {
-        const { page } = context;
-        if (page !== 0) {
-          return {
-            options: [],
-          };
-        }
-
-        const { sobjects } = await this.salesforce.listSObjectTypes();
-        const options = sobjects
-          .filter(this.isValidSObject)
```

Comment:
Removing this will stop querying Salesforce for the list of available objects in the user's instance. Is this the intention here?

Moreover, querying for updates for some of the objects in Salesforce might not be possible if those objects don't have the `replicateable` flag set (that's what the `.filter` does).

Keep in mind that this particular module/file does not use the webhook approach, so I'm not sure we should even touch this file (unless you have more context and I'm missing something 🤔).

Reply:
The `propDefinition` part makes it inherit the `async options()` method from the app. The app code is the same as this removed part.

Since you've mentioned this, I've noticed and changed back the label and description for the prop.

Reply:
I see, so can we also move the `isValidSObject` method to the app module, and replace the hardcoded lambda function in the `.filter` with a call to that method? I'll leave a suggestion in the `.app.mjs` file to reflect this.